test/api/geocoding.test.js

```diff
@@ -5,167 +5,77 @@ const sinonChai = require('sinon-chai');
 chai.use(sinonChai);
 const { expect } = chai;
 
-import * as Http from '../../src/http';
+import { latitude, longitude } from '../common';
+
+import Http from '../../src/http';
 import Navigator from '../../src/navigator';
-import STATUS from '../../src/status_codes';
+import ERROR from '../../src/error_codes';
 
 import Geocoding from '../../src/api/geocoding';
 
 describe('Geocoding', () => {
   let httpStub;
   let navigatorStub;
 
-  const latitude = 40.7041895;
-  const longitude = -73.9867797;
+  const query = '20 Jay Street';
 
-  const mockQuery = '20 Jay Street';
+  const geocodeResponse = { meta: {}, address: {} };
```

Comment:
nit: forward / reverse return `addresses`, ip returns `address` - fine to leave as is since it's just testing though